### PR TITLE
Add missing tests for record classes and localized-select attribute type

### PR DESCRIPTION
**Pull request by flagbird**

### CODE REVIEW

**Comment by mralexandernickel** [comment]
@flagbird Why do you use `.tsx` as extension everywhere? To me this is looking like regular typescript 🤷

**Comment by flagbird** [comment]
@mralexandernickel Because the usage of `<componentName>` tags run into errors when it's not a tsx file.

File: jest/public/reference-entity/record/table/view.test.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+    test('ValueUpdater invokes delete-row on click', function () {
+        const onchange = jest.fn();
+        const renderedView = renderView(onchange);
+
+        const removeButton = renderedView.find('td').at(3).children();
+
+        global.confirm = () => {
+            return true;
+        };
```
Comment (mralexandernickel) [inline jest/public/reference-entity/record/table/view.test.tsx:99]: Could be shortened to
`global.confirm = () => true;`

**Review by mralexandernickel — APPROVED** [review]

**Comment by flagbird** [comment]
Thanks for the review.

**Pull request merged**